velox/exec/OutputBuffer.cpp

```diff
@@ -474,7 +474,7 @@ bool OutputBuffer::enqueue(
         VELOX_UNREACHABLE(PartitionedOutputNode::kindString(kind_));
     }
 
-    if (bufferedBytes_ > maxSize_ && future) {
+    if (bufferedBytes_ >= maxSize_ && future) {
```

Comment:
nit: is this a bug fix? perhaps, mention in the PR description.

Reply:
It's hard to call it a bug. I doubt the `bufferedBytes` lands right at the boundary very often in real life. It's just more convenient to have it "equal" than "greater than" when writing tests.